src/app/pages/layouts/top-bar/top-bar.tsx

```diff
@@ -10,8 +11,13 @@ export class TopBar extends Component {
                     <TopAppBar.Section align-start>
                         <img class="top-appbar-icon" src="/assets/logo.svg"></img>
                     </TopAppBar.Section>
+                    <TopAppBar.Section>
```

Comment:
Using JSX does not equals html output. So, we should _always_ use self-closing tags, if there's no content inside.
```js
<FooComponent baz={false}></FooComponent>

// instead use this:
<FooComponent baz={false} />
```